src/e2e/specs/landing.spec.js

```diff
@@ -59,3 +59,54 @@ test.describe('Landing Page element verification', () => {
     })
   })
 })
+
+test.describe('Landing Page Functionality Verification', () => {
+  test.beforeEach(async ({ landingPage }) => {
+    await landingPage.open()
+  })
+
+  test('Verify landing page elements - free scan @smoke', async ({ landingPage, scanPage, page }) => {
```

Comment:
I thought old pages weren't going to be part of the smoke tests? (The landing page should be taken over by Bedrock.)

```suggestion
  test('Verify landing page elements - free scan', async ({ landingPage, scanPage, page }) => {
```

Reply:
ah ok, it was talked about but never heard any confirmations on this, will remove it from smoke